h/presenters/organization_json.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+
+class OrganizationJSONPresenter(object):
+    """Present an organization in the JSON format returned by API requests."""
+
+    def __init__(self, organization_resource):
+        self.resource = organization_resource
+        self.organization = organization_resource.organization
+
+    def asdict(self):
+        model = self._model()
+        self._logo(model)
+        return model
+
+    def _model(self):
+        model = {
+          'name': self.organization.name,
+          'id': self.organization.pubid,
+        }
+        return model
+
+    def _logo(self, model):
+        if self.resource.logo is not None:
+            model['logo'] = self.resource.logo
```

Comment:
We have an existing precedent (eg. in `/api/profile`) of using `null` rather than omitting properties to indicate missing optional fields. Even though it doesn't matter for JavaScript in many cases, I still think it would be better to do so here for the benefit of clients in other languages.

I know we opted not to do it for the `organization` property in the `/api/groups` response but my understanding is that is because we were planning to make that field mandatory.

Reply:
I gave this some thought, and I think I agree with you. Especially in this case, where the value and even type of `organization` varies depending on the request—we need to give more explicit feedback.

In the case of a request with `expand` containing `organizations`, I am definitely seeing the light that we should return a `null` here to represent that the expanded field doesn't have anything to expand (but that we "heard" the client's request for it to be expanded). This is semantically more better. So I will make this change.

I think we still have some definition work on how to represent this value when it's not expanded (I opened another note/card to this effect). We can tackle that separately.